themes/ddbasic/sass/components/messages.scss

```diff
@@ -0,0 +1,77 @@
+@import '../base.scss';
```

Comment:
Describe component contents/purpose, why are these styles grouped? What is the common denominator?